codegen/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "loopback-sdk-ios-codegen",
```

Comment:
I have a bit of hesitation to do so, since those modules are required by things under the codegen directory only...  It'd be great if we could have a chat about it.